LoopKitUI/View Controllers/DismissableHostingController.swift

```diff
@@ -10,16 +10,41 @@ import SwiftUI
 
 
 public class DismissibleHostingController: UIHostingController<AnyView> {
+    public enum DismissalMode {
+        case modalDismiss
+        case pop(to: UIViewController.Type)
```

Comment:
Definitely would love to have this for the settings editors...so that it can navigate back upon hitting "Save"... but why do you need the type?  I'd expect "pop" would just be "go back to wherever you were before")?

Reply:
It would be simple enough to add an additional case to this enum (e.g. `popToPrevious`) to go back to the previous controller if you need that behavior. UINavigationController supports popping back to an arbitrary place in the navigation chain, and my use case requires going back multiple controllers.

(Tangential: the settings editors were designed to be modals to make cancellation explicit, no? Did that design change?)

Reply:
@mpangburn design is still in flux, but one of the discussed behaviors was to go "back" after the user tapped "Save".  (The current "latest" thinking is it would also present a passcode challenge)